# HG changeset patch
# User Rik <rik@octave.org>
# Date 1516759524 28800
# Node ID 3d78a7eb5aa466a1afd11593710a67b4e183c4c1
# Parent  9d01ce02d5cb7d16efbaf0a585de5d2e3596990c
Make "PickableParts" property for figures the fixed value of "visible".

* graphics.in.h: Override base_properties "pickableparts" with figure
object "pickableparts" property which is hidden and only supports
the value "visible".

diff --git a/libinterp/corefcn/graphics.in.h b/libinterp/corefcn/graphics.in.h
--- a/libinterp/corefcn/graphics.in.h
+++ b/libinterp/corefcn/graphics.in.h
@@ -3159,6 +3159,12 @@
       callback_property windowkeyreleasefcn , Matrix ()
       callback_property windowscrollwheelfcn , Matrix ()
       radio_property windowstyle , "{normal}|modal|docked"
+
+      // Overridden base property
+      // Property is not implemented for figures.
+      // Hide it and set it to a default value that works.
+      radio_property pickableparts h , "{visible}"
+
       // Octave-specific properties
       mutable string_property __gl_extensions__ hr , ""
       mutable string_property __gl_renderer__ hr , ""

